security/Security.php

```diff
@@ -95,7 +95,7 @@ class Security extends Controller {
 	 *
 	 * @var array|string
```

Comment:
Hmm, so why does having it as empty string cause any problems?!

I think this should just be null / undefined, tbh.

Can I ask you to mark it `@config` please?